starfish/_display.py

```diff
@@ -108,6 +108,7 @@ def _spots_to_markers(intensity_table: IntensityTable) -> Tuple[np.ndarray, np.n
 def display(
         stack: Optional[ImageStack] = None,
         spots: Optional[IntensityTable] = None,
+        viewer=None,
```

Comment:
Is it possible to give a type hint here without importing `Viewer` from `napari`?

Reply:
It seems to me like we are trying to avoid importing napari at the top of _display.py to ease the dependency. Is this the case?

Reply:
Sort of. The `try-except` block can be moved up to the top but I didn't at the time because the warning would've been generated every time on start-up.

I suppose you could instead put the following at the top:
```python
try:
    from napari import Window, Viewer
except ImportError:
    Window, Viewer = None, None
```

and replace the warning code at the bottom with:
```python
if Window is None or Viewer is None:
    warnings.warn(...)
    return
```

Reply:
Sounds good. I'll do this. Thanks!